README.md

```diff
@@ -0,0 +1,65 @@
+# Spendy
+
+Spendy is fullstack web appliactions that helps users to track purchases.
+
+## Built With
+
+**Languages**
+
+- Ruby
+- SQL
+
+**Frameworks**
+
+- Ruby on Rails
+- Postgresql
+
+## Live Demo
+
+- [Demo](https://mighty-refuge-76823.herokuapp.com/)
+
+## Video Presentation
+
+- [Video](https://drive.google.com/file/d/1RvsTJASZ36tnAQYaLLvOs5ZcyPiaD8dm/view?usp=sharing)
+
+## Getting Started
+
+**To get a local copy of the project**
+
+- Type `git clone https://github.com/afizsavage/spendy.git` to clone the repository
+- Type `cd spendy` to move to the project directory
+- Type `bundle install` to install project dependencies
+
+**To Run Migration files**
+
+- - Type `rails db:migrate` to run migration files
+
+**To run development environment and build tailwindcss files**
+
+- - Type `./bin/dev` to run development environment
+
+**To run the project**
+
+- Type `rails s` to run the project
+
+## Run Tests
+
+Type `bundle exec rspec` in the command line to run the tests
+
+## Author
+
+👤 **Afiz Savage**
+
+- GitHub: [@afizsavage](https://github.com/afizsavage)
+- Twitter: [@fizzo_geek](https://twitter.com/fizzo_geek)
+- LinkedIn: [@afiz-savage](https://www.linkedin.com/in/afiz-savage-3b91a21ba/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) but apart from that your app is working fine . Please make sure to fix that.